templates/quiz.html

```diff
@@ -0,0 +1,123 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="static/styles.css">
+  <script src="static/main.js" type="module"></script>
+  <title>AnimalQuiz</title>
+</head>
+
+<body>
+  <div 
+    id="switch-class"
+    x-data="{ animalclass: $store.animalclass }"

```

Comment:
Try:
```suggestion
    x-data="{ animalclass: $store.data.animalclass }"

```
...that's where it's being stored in `main.js`!

Reply:
Oh dear, that was a silly one... :)